styles/gauntlet.css

```diff
@@ -63,6 +69,22 @@ SELECT LAYOUT STYLES
 	margin-bottom: 20px;
 }
 
+#species-select[disabled],
+#class-select[disabled],
+#weapon-select[disabled],
+input[disabled] {
+	background-color: lightgrey;
+}
```

Comment:
For some reason on my computer, when these are disabled they are bigger than they are when the are enabled, and they throw off the formatting.